packages/fast-usdc/src/exos/settler.js

```diff
@@ -199,12 +209,17 @@ export const prepareSettler = (
         ) {
           const { mintedEarly } = this.state;
           const { value: fullValue } = fullAmount;
+          // XXX i think this only contains Advancing txs - should this be a separate store?
           const key = makeMintedEarlyKey(forwardingAddress, fullValue);
           if (mintedEarly.has(key)) {
             mintedEarly.delete(key);
             if (success) {
+              // TODO: does not write `ADVANCED` to vstorage
```

Comment:
TODO when? in this PR?

Reply:
#10750 but maybe will collapse into this

Reply:
Closing 10750 and collapsing into this PR